text/0000-2023H2-roadmap.md

```diff
@@ -0,0 +1,177 @@
+
+As a project, we've made good progress on our H1 roadmap, but still need some more time to get some of them across the finish line.
+
+#### Release Base Image Extension
+* Owner: @natalieparellano
+* Status: Continue
+* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0105-dockerfiles.md)
+
+This is close and will be released as part of Platform `0.12`, Buildpack `0.10`, lifecycle `0.17.0`, and pack `0.30.0`.
+
+#### Remove Stacks & Mixins
+* Owner: @jkutner
+* Status: Continue
+* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)
+
+This is close and will be released as part of Platform `0.12`, Buildpack `0.10`, lifecycle `0.17.0`, and pack `0.30.0`.
+
+#### Execution Environments RFC
+* Owner: @hone
+* Status: Continue
+* Links: [RFC](https://github.com/buildpacks/rfcs/pull/274)
+
+The RFC is written, but feedback needs to be incoporated before re-opening for review.
+
+#### Project Health
+* Owner: @samj1912
+* Status: Finished
+* Links: [Buildpacks Community Organization RFC](https://github.com/buildpacks/rfcs/pull/273)
+
+The RFC has been merged and the buildpacks community GitHub org have been created.
+
+#### Pack Test Cleaning/Optimizations
+* Owner: @dfreilich
+* Status: Defer
+* Links: [Pack Pull Request](https://github.com/buildpacks/pack/pull/1498)
+
+This item has been deferred for now, but we will work with anyone who wants to push this forward.
+
+### New Items
+
+#### Community Engagement Health Checks
+* Owner: @microwavables (Team Lead sponsor @jkutner)
+* Links: [VMware Tanzu Community Engagement Health Checks](https://github.com/vmware-tanzu/community-engagement/blob/main/HEALTHCHECKS.md)
+
+The project is lucky to have a Community Manager! This is one of the projects proposed by @microwavables to set a base line to measure how we're doing as a community.
+
+#### RFC for Buildpack Author Observability
+* Owner: @joshwlewis (Team Lead sponsor @hone)
+* Links: TBD
+
+Currently, Buildpack Authors have little to no tools around visibility with their buildpacks as they run on a platform, including `pack`. In some of the Heroku v2 buildpacks, they implemented logging that could be handed off to the platform by running `bin/report`. This work stream is about standardizing output for both successful AND failed builds that Buildpack Authors can use to instrument their buildpack.
+
+#### Private Registry Mirrors
+* Owner: @jabrown85
+* Links: [RFC](https://github.com/buildpacks/rfcs/pull/285)
+
+A platform operator can configure registry mirrors that lifecycle could use without needing for the manifest to have to point to it. This will allow a platform to reduce the risk of service operations from external registry sources and reduce public network bandwidth usage. The resulting image when taken off platform will also function without needing access to the registry mirror.
+
+#### kpack Donation
+* Owner: @jjbustamante (Team Lead sponsor @hone)
```

Comment:
@hone would you be okay with this being me? 
```suggestion
* Owner: @jjbustamante (Team Lead sponsor samj1912)
```

Reply:
@samj1912 :+1: